types/index.d.ts

```diff
@@ -283,4 +283,4 @@ declare namespace ElementUI {
   export class Upload extends ElUpload {}
 }
 
-export default ElementUI
+export = ElementUI
```

Comment:
Hi, could you please tell me why this line is changed?

Reply:
Hi, this was a mistake, I'm removing this change.